hack/test-cri.sh

```diff
@@ -61,6 +61,7 @@ if [ ! -x "$(command -v containerd)" ]; then
 fi
 sudo pkill containerd
 sudo containerd -l debug &> ${REPORT_DIR}/containerd.log &
+sleep 1 # sleep 1 seconds for containerd to be ready.
```

Comment:
I tried to put a look to periodically check `/run/containerd/containerd.sock` existence, but I find that I need root permission to do that, which makes things a little bit complex. So just sleep for 1 sec for now.